regression.sh

```diff
@@ -10,38 +16,40 @@ heading() {
 
 FD1="./fd-master"
 FD2="./fd-feature"
+RESULT_DIR="regression-results"
 
-rm -f regression-results/*.md
+mkdir -p "$RESULT_DIR"
+rm -f $RESULT_DIR/*.md
```

Comment:
I always get a little bit nervous when I see `rm` with unquoted variables. Just to be safe (if someone changes `RESULTS_DIR` to `my folder`, for example), we should quote it:
```bash
rm -rf "$RESULTS_DIR"/*.md
```

Reply:
Fair enough - I actually wasn't aware you could partially quote a bash expansion, so :+1: to learning things